mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksViewModel.kt

```diff
@@ -56,7 +51,7 @@ abstract class MavericksViewModel<S : MavericksState>(
 
     private val stateStore = config.stateStore
     private val lastDeliveredStates = ConcurrentHashMap<String, Any>()
-    private val activeSubscriptions = Collections.newSetFromMap(ConcurrentHashMap<String, Boolean>())
+    private val activeSubscriptions: MutableSet<String> = mutableSetOf()
```

Comment:
why remove the concurrency?

Reply:
oh oops, didnt realize newSetFromMap kept it backed with the map you gave it. reverted